tests/ops/models/test_privacy_request.py

```diff
@@ -392,7 +392,7 @@ def test_trigger_two_way_policy_webhook_add_derived_identity(
             privacy_request.trigger_policy_webhook(webhook)
             db.refresh(privacy_request)
             assert privacy_request.status == PrivacyRequestStatus.in_processing
-            assert privacy_request.get_cached_identity_data() == {
+            assert privacy_request.get_identity_map() == {
                 "email": "customer-1@example.com",
                 "phone_number": "+5555555555",
```

Comment:
This existing assertion is justification for the order of provided and derived identities in `get_identity_map`